qa/tasks/deepsea_deploy.py

```diff
@@ -41,6 +41,11 @@
    /srv/salt/ceph/stage/prep/minion/default.sls
 """
 global_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/global.conf'
+mon_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/mon.conf'
+mgr_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/mgr.conf'
+mds_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/mds.conf'
+osd_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/osd.conf'
+client_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/client.conf'
 health_ok_cmd = "health-ok.sh --teuthology"
 mon_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/mon.conf'
```

Comment:
This is the duplicate line - it could be removed.

Reply:
This I did because how it's stated in deepsea /srv/salt/ceph/configuration/files/ceph.conf.d/README

> The default ceph.conf template
> (<salt_dir>/ceph/configuration/files/ceph.conf.j2) includes the content of six
> files from the <salt_dir>/ceph/configuration/files/ceph.conf.d directory. These
> files are [global,mon,mgr,mds,osd,client].conf and are meant for user
> customisations of ceph.conf.